src/room/track/RemoteTrack.ts

```diff
@@ -27,15 +28,18 @@ export default abstract class RemoteTrack extends Track {
   initializeEncryption(password: string) {
     this.setPassword(password);
     this.decryptTrack();
+    setInterval(() => {
+      this.worker.postMessage({ operation: 'rotateKey' });
+    }, 20000);
```

Comment:
it should be a defined constant and the reason it's not is because there's a change that got left out of that commit :facepalm:  

thanks for the catch